.github/workflows/test.yml

```diff
@@ -29,3 +29,38 @@ jobs:
 
       - name: Run tests
         run: make test
+
+  functional_test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6]
+        script:
+          - scripts/generate-assessment-schema.py g-cloud-12
+          - scripts/generate-assessment-schema.py digital-outcomes-and-specialists-5
+          - scripts/generate-search-config.py g-cloud-12 services --output-path=/tmp
+          - scripts/generate-search-config.py digital-outcomes-and-specialists-5 briefs --output-path=/tmp
+          - scripts/generate-validation-schemas.py --output-path=/tmp
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Ubuntu packages
+        run: sudo apt-get install libxml2-dev libxslt1-dev python-dev
```

Comment:
Do we need this? I don't think this repo uses `lxml`.

Reply:
Not sure, I just copied it from above...

Reply:
Here's one way to find out https://github.com/alphagov/digitalmarketplace-frameworks/pull/660/commits/8935b13b54b1b48a35aab5ca0994a4ef147e0946...

Reply:
It worked! 😁